src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/BorderTests/Border_Tests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.BorderTests
+{
+	public class Border_Tests : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void Change_Manipulation_Property()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.BorderTests.Border_Clipped_Change_Property");
+
+			_app.WaitForElement("ClippedBorder");
+
+			_app.Tap("ClippedBorder");
+
+			_app.WaitForDependencyPropertyValue(_app.Marked("ClippedBorder"), "ManipulationMode", "None");
+
+			TakeScreenshot("After property change");
```

Comment:
Should we validate that some pixel is of some color after the change ?

Reply:
Good idea! I'll add a check that before/after are equal